src/guides/v2.3/install-gde/install/cli/dev_options.md

```diff
@@ -1,6 +1,6 @@
 ---
 group: installation-guide
-title: Contributing developers&mdash;update, reinstall Magento
+title: Contributing developers—update, reinstall Magento
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**